lib/fork.js

```diff
@@ -86,7 +86,7 @@ module.exports = function (file, opts) {
 		ps.on('results', function (data) {
 			results = data;
 			data.tests = testResults;
-			send(ps, 'teardown');
+			send('teardown');
```

Comment:
I don't agree. Removal of one argument does not make things any simpler in this case. I'd prefer to leave it to make it more clear. It speaks for itself, "send ps a teardown event". 

While `send('teardown')` leaves me (at least) wondering where is this getting sent, what is the context.

If you are worried about consistency, we can compromise by using `send(process, 'some-event')` in the child processes to send event to a parent process.